packages/compass-e2e-tests/helpers/commands/export-to-language.ts

```diff
@@ -56,15 +56,10 @@ export async function exportToLanguage(
     }
   }
 
-  let text = '';
-
-  if (process.env.COMPASS_E2E_DISABLE_CLIPBOARD_USAGE === 'true') {
-    text = await browser.$(Selectors.ExportToLanguageQueryOutput).getText();
-  } else {
-    // copy the text to and from the clipboard so we can return it later
-    await browser.clickVisible(Selectors.ExportToLanguageCopyOutputButton);
-    text = await clipboard.read();
-  }
+  // We are not testing the leafygreen copy button here because clicking it (and
+  // probably the follow-up tooltip that shows up) breaks clicking the close
+  // buttons in the modal and it's hard to coordinate the flow to avoid
+  const text = await browser.$(Selectors.ExportToLanguageQueryOutput).getText();
```

Comment:
Not feeling that great about it, but the other option of trying to catch the copy button tooltip to show up and disappear was super flaky and I'd rather not introduce flakes like that. This is leafygreen component behavior, so not testing it in our e2e tests seems like a reasonable compromise